ci/Dockerfile.lambda

```diff
@@ -32,5 +33,8 @@ COPY bin/entry.sh /
 COPY bin/sync_lambda_envs.sh /
 RUN chmod 755 /usr/bin/aws-lambda-rie /entry.sh /sync_lambda_envs.sh
 
+# New Relic lambda layer
+RUN unzip newrelic-layer.zip -d /opt && rm newrelic-layer.zip
+
 ENTRYPOINT [ "/entry.sh" ]
-CMD [ "application.handler" ]
\ No newline at end of file
+CMD [ "newrelic_lambda_wrapper.handler" ]
```

Comment:
Can we please add a comment about how this works and how this is configured via an environment variable?